modules/nextflow/build.gradle

```diff
@@ -41,14 +44,13 @@ dependencies {
     api "commons-lang:commons-lang:2.6"
     api "commons-codec:commons-codec:1.15"
     api "commons-io:commons-io:2.15.1"
-    api "com.beust:jcommander:1.35"
     api("com.esotericsoftware.kryo:kryo:2.24.0") { exclude group: 'com.esotericsoftware.minlog', module: 'minlog' }
-    api('org.iq80.leveldb:leveldb:0.12')
-    api('org.eclipse.jgit:org.eclipse.jgit:7.1.1.202505221757-r')
-    api ('javax.activation:activation:1.1.1')
-    api ('javax.mail:mail:1.4.7')
-    api ('org.yaml:snakeyaml:2.2')
-    api ('org.jsoup:jsoup:1.15.4')
+    api 'org.iq80.leveldb:leveldb:0.12'
+    api 'org.eclipse.jgit:org.eclipse.jgit:7.1.0.202411261347-r'
```

Comment:
Was this downgraded on purpose? before it was `7.1.1.202505221757`

Reply:
this is likely an incorrect merge commit, none of the versions should be changed. I will fix it